src/deploy/hosting/release.ts

```diff
@@ -17,11 +18,12 @@ export async function release(context: any, options: any): Promise<void> {
   await Promise.all(
     context.hosting.deploys.map(async (deploy: any) => {
       utils.logLabeledBullet(`hosting[${deploy.site}]`, "finalizing version...");
-      const finalizeResult = await client.patch(
-        `/${deploy.version}`,
-        { status: "FINALIZED" },
-        { queryParams: { updateMask: "status" } }
-      );
+
+      const config = convertConfig(deploy.config);
```

Comment:
This is expanding scope from your previous work, but I wonder if we should also make `convertConfig` take the functions payload and detect that a function rewrite was a GCFv2 function and turn it into a Run rewrite. That'll fix the edge case where people must know to use the service rewrite for gen 2 functions until July.

Reply:
I don't think that's a big lift from what I've already built here, happy to take a swing.